src/components/BossesPanel.js

```diff
@@ -1,181 +1,109 @@
-                                />
-                            ) : (
-                                <td />
-                            )}
-                            {i * 5 + 2 < BOSSES.length ? (
-                                <BossTile
-                                    boss={BOSSES[i * 5 + 2]}
-                                    selectedBoss={selectedBoss}
-                                    setSelectedBoss={setSelectedBoss}
-                                    unlockedBosses={unlockedBosses}
-                                    characterStats={hiscores}
-                                />
-                            ) : (
-                                <td />
-                            )}
-                            {i * 5 + 3 < BOSSES.length ? (
-                                <BossTile
-                                    boss={BOSSES[i * 5 + 3]}
-                                    selectedBoss={selectedBoss}
-                                    setSelectedBoss={setSelectedBoss}
-                                    unlockedBosses={unlockedBosses}
-                                    characterStats={hiscores}
-                                />
-                            ) : (
-                                <td />
-                            )}
-                            {i * 5 + 3 < BOSSES.length ? (
-                                <BossTile
-                                    boss={BOSSES[i * 5 + 4]}
-                                    selectedBoss={selectedBoss}
-                                    setSelectedBoss={setSelectedBoss}
-                                    unlockedBosses={unlockedBosses}
-                                    characterStats={hiscores}
-                                />
-                            ) : (
-                                <td />
-                            )}
-                        </tr>
-                    ))}
-                </tbody>
-            </table>
-            <div className='w-full px-1 mt-2 align-top'>
-                {selectedBoss ? (
+    const renderBossDetails = () =>
+        selectedBoss ? (
+            <>
+                <p className='heading-accent-md'>{selectedBoss.label}</p>
+                {hiscores && unlockedBosses.includes(selectedBoss.label) && (
                     <>
-                        {/* TODO add sage's renown values for unlocks */}
-                        {/* <span className='heading-accent-md'>{selectedBoss}</span>
-                        <p className='italic text-sm'>Cost: 500 league points</p> */}
+                        <p className='italic text-sm'>
+                            Kills: {numberWithCommas(hiscores[selectedBoss.hiscoresName].score)}
+                        </p>
+                        <p className='italic text-sm'>
+                            Rank: {numberWithCommas(hiscores[selectedBoss.hiscoresName].rank)}
+                        </p>
```

Comment:
do we need any fallbacks here if you don't have any kc? Or does it have a default already

Reply:
Fair point, yes i expect it will show -1 so i'll change it to be a `-`